bsd/backends.py

```diff
@@ -3,6 +3,9 @@
 from xml.etree.ElementTree import ElementTree
 from .api import BSD
 from .models import BSDProfile
+import logging
+
+logger = logging.getLogger(__name__)
```

Comment:
Yeah I wonder what you think about making logger available as a standard? I notice that during development I often add the logger to help test, then take it out. Would it be better to just always have it available? It would also encourage more use of logging which is probably good practice for us.

But I agree as a general rule better to not include things that are not in current use.